velox/exec/tests/TableWriteTest.cpp

```diff
@@ -3441,20 +3441,39 @@ DEBUG_ONLY_TEST_F(TableWriterArbitrationTest, reclaimFromTableWriter) {
                   {fmt::format(
                       "sum({})", TableWriteTraits::rowCountColumnName())})
               .planNode();
-
-      AssertQueryBuilder(duckDbQueryRunner_)
-          .queryCtx(queryCtx)
-          .maxDrivers(1)
-          .spillDirectory(spillDirectory->path)
-          .config(core::QueryConfig::kSpillEnabled, writerSpillEnabled)
-          .config(core::QueryConfig::kWriterSpillEnabled, writerSpillEnabled)
-          // Set 0 file writer flush threshold to always trigger flush in test.
-          .config(core::QueryConfig::kWriterFlushThresholdBytes, 0)
-          .plan(std::move(writerPlan))
-          .assertResults(fmt::format("SELECT {}", numRows));
-
-      ASSERT_EQ(arbitrator->stats().numFailures, writerSpillEnabled ? 0 : 1);
-      ASSERT_EQ(arbitrator->stats().numNonReclaimableAttempts, 0);
+      {
+        auto task =
+            AssertQueryBuilder(duckDbQueryRunner_)
+                .queryCtx(queryCtx)
+                .maxDrivers(1)
+                .spillDirectory(spillDirectory->path)
+                .config(core::QueryConfig::kSpillEnabled, writerSpillEnabled)
+                .config(
+                    core::QueryConfig::kWriterSpillEnabled, writerSpillEnabled)
+                // Set 0 file writer flush threshold to always trigger flush in test.
+                .config(core::QueryConfig::kWriterFlushThresholdBytes, 0)
+                .plan(std::move(writerPlan))
+                .assertResults(fmt::format("SELECT {}", numRows));
+
+        auto opStats = task->taskStats().pipelineStats.front().operatorStats;
```

Comment:
Can you see the example of using toPlanStats to get operator stats based on the plan node id? Thanks!